src/main/java/io/quarkus/registry/app/model/ExtensionRelease.java

```diff
@@ -105,9 +108,14 @@ public static Optional<ExtensionRelease> findByGAV(String groupId, String artifa
     }
 
     public static List<ExtensionRelease> findNonPlatformExtensions(String quarkusCore) {
+        DefaultArtifactVersion artifactVersion = new DefaultArtifactVersion(quarkusCore);
+        String minimumVersion = artifactVersion.getMajorVersion() + ".0.0.Final";
+        String maximumVersion = (artifactVersion.getMajorVersion() + 1) + ".0.0.Final";
```

Comment:
The upper bound should also be Alpha for the same reason.

Reply:
Ah good catch, will do that